maptivist/src/main/webapp/script.js

```diff
@@ -32,13 +32,74 @@ function initMap(){
     mapId: '837a93b1537b2a61'
   });
 
+  // EXAMPLE MARKER
```

Comment:
Do we still need this example marker?

Reply:
No it's taken off in the master now so it can be removed here too